ws_web/public_views.py

```diff
@@ -403,6 +403,31 @@ def create(self, request, *args, **kwargs):
             )
             serializer = TagsSerializer(data=data_to_save, many=True)
             if serializer.is_valid():
+
+                # Test to make sure that the sense IDs are reasonable
+                # Especially that they are related to the gloss_with_replacement word
+                # get the pos so we can lemmatize
+                queryset = DerivedTokens.objects.filter(
```

Comment:
probably consider putting this whole block into a try/except to avoid at least runtime error's blocking saving events.

Reply:
done